package/gluon-mesh-vpn-wireguard/README.md

```diff
@@ -0,0 +1,38 @@
+# gluon-mesh-vpn-wireguard
+
+This package will allow wireguard [1] to be used in gluon. Wireguard being a
+VPN on OSI layer 3 is useful for mesh protocols that operate on layer 3.
+
+[1] https://wireguard.io
+
+## including servers via site.conf
+This is similar to the fastd-based mesh_vpn structure.
+
+```
+          mesh_vpn = {
+                  mtu = 1374,
+
+                  wireguard = {
+                          enabled = true,
+                          groups = {
+                                  backbone = {
+                                          limit = '2', -- currently unused
```

Comment:
I think it is no problem for WireGuard to have a lot of peers (at least this was one of the design goals).
Thus do we need a limit here and I think we should not introduce a field that is not used atm.

Reply:
it should be possible to specify n peers but only connect to m (0 < m < n) of them. That is what the same logic as for fastd. The comment "currently unused" is simply outdated and was removed.